# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/openjdk/acceptable-bootjdk-versions.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- jdk21u-jdk-21.0.3-ga/make/conf/version-numbers.conf.vanilla	2024-11-24 19:51:05.979922076 +0100
+++ jdk21u-jdk-21.0.3-ga/make/conf/version-numbers.conf	2024-11-24 19:51:13.473255288 +0100
@@ -37,6 +37,6 @@
 DEFAULT_VERSION_CLASSFILE_MAJOR=65  # "`$EXPR $DEFAULT_VERSION_FEATURE + 44`"
 DEFAULT_VERSION_CLASSFILE_MINOR=0
 DEFAULT_VERSION_DOCS_API_SINCE=11
-DEFAULT_ACCEPTABLE_BOOT_VERSIONS="20 21"
+DEFAULT_ACCEPTABLE_BOOT_VERSIONS="20 21 22 23"
 DEFAULT_JDK_SOURCE_TARGET_VERSION=21
 DEFAULT_PROMOTED_VERSION_PRE=
